src/main/java/org/apache/commons/fileupload/FileUploadBase.java

```diff
@@ -205,6 +205,18 @@ public static boolean isMultipartContent(HttpServletRequest req) {
     public long getSizeMax() {
         return sizeMax;
     }
+    
+    /**
+     * This method enables subclassed to set 
+     allowed size per request if needed.
+     * @param ctx
+     * @return
+     */
```

Comment:
Fill in missing tags. Add since tag.

Reply:
What is the role of the `ctx` parameter?

Reply:
ctx can be used to get request attributes which can be used in calculating allowed max size for particular request.